sdk/cosmos/microsoft-azure-cosmos/src/main/java/com/azure/data/cosmos/internal/changefeed/implementation/LeaseStoreManagerImpl.java

```diff
@@ -141,7 +141,7 @@ public LeaseStoreManagerBuilderDefinition hostName(String hostName) {
 
     @Override
     public Mono<Lease> createLeaseIfNotExist(String leaseToken, String continuationToken) {
-        if (leaseToken == null) throw new IllegalArgumentException("leaseToken");
+        if (leaseToken == null) throw Exceptions.propagate(new IllegalArgumentException("leaseToken"));
```

Comment:
you don't need `Exception.propagate(.)` here.

`Exception.propagate(.)` will wrap the exception in runtime exception if it is not already a runtime exception.
In this case `IllegalArgumentException` is already a runtime exception.